cmd/libs/go2idl/go-to-protobuf/protobuf/generator.go

```diff
@@ -258,6 +258,7 @@ func (b bodyGen) doStruct(sw *generator.SnippetWriter) error {
 					options = append(options, fmt.Sprintf("%s = %s", key, v))
 				}
 			}
+		// TODO: What's that? What's the purpose of it?
```

Comment:
Was historical before I dropped TypeMeta from the serialization.  Basically, allows you to define that a field should be directly embedded